packages/app-store/salesforce/lib/CrmService.ts

```diff
@@ -328,6 +334,7 @@ export default class SalesforceCRMService implements CRM {
       (forRoundRobinSkip ? appOptions?.roundRobinSkipCheckRecordOn : appOptions?.createEventOn) ??
       SalesforceRecordEnum.CONTACT;
     let soql: string;
+    let accountOwnerId = "";
```

Comment:
Fixing a bug I found here. If we're creating events on contacts and then leads, but we're looking up account ownership, we were accidently returning the contact owner instead of the account owner.